lib/data/models/user_model.dart

```diff
@@ -1,16 +1,19 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:equatable/equatable.dart';
+import 'package:nowingoogle/domain/entities/user.dart';
 
 class UserModel extends Equatable {
   final String firstName;
   final String lastName;
   final String username;
   final String email;
-  final String phone;
+  final String? phone;
   final int age;
   final String gender;
   final String place;
   final String career;
   final String organization;
+  final String avatar;
```

Comment:
is avatar taken from user's google login? or can they manually add/upload avatars?

Reply:
Nope anna, here when we meant avatar, we wanted the avatar to be inclusive at the time, be part of the platform, we were going for something like bitmoji!

Reply:
> Nope @Harryalways317 , here when we meant avatar, we wanted the avatar to be inclusive at the time, be part of the platform, we were going for something like bitmoji! So, here avatar refers to the id of the avatar.

We might offer dynamic personalized avatars going further!

Reply:
Thats nice then avatar is stored as a reference that has id and avatar_url, so we may need to tweak the model once we got there